js/README.md

```diff
@@ -26,5 +26,6 @@ implemented in a language that fully supports server-side functionality, such as
 | [Ice Context](./Ice/context/)             | Shows how to set and retrieve request contexts.                                                     |
 | [Ice Filesystem](./Ice/filesystem/)       | Shows the power of interface inheritance in Slice.                                                  |
 | [Ice Greeter](./Ice/greeter/)             | Shows how to call and implement a canonical Greeter application with Ice. **Start with this demo!** |
+| [IceStorm Weather](./IceStorm/Weather/)   | Shows how to use IceStorm to create a simple pub-sub application.                                   |
```

Comment:
Are all these demos really Typescript demos?

Reply:
Yes, we have:

```
./Ice/cancellation
./Ice/context
./Ice/greeter
./Ice/fileSystem
./Ice/bidir
./Glacier2/callback
./IceStorm/weather
```

All code is TypeScript, compiled to JavaScript and target Node.js